sembast/doc/queries.md

```diff
@@ -139,14 +139,25 @@ var finder = Finder(filter: Filter.equals('product.code', 'AF8'));
 
 ### Composite filter
 
-You can combine multiple filters using `Filter.or` or `Filter.and`
+You can combine multiple filters using the operators `&` and `|`:
```

Comment:
What about operator precedence?
`filter1 & filter2 | filter3` vs `filter2 | filter3 & filter1`

Reply:
[Dart's precedence rules](https://dart.dev/guides/language/language-tour#operators) specify `&` to have higher precedence than `|`, hence your snippets could be written as `(filter1 & filter2) | filter3` and `filter2 | (filter3 & filter1)`, respectively.